test/gtest/common/mem_buffer.cc

```diff
@@ -239,6 +239,12 @@ void *mem_buffer::allocate(size_t size, ucs_memory_type_t mem_type)
 #if HAVE_CUDA
     case UCS_MEMORY_TYPE_CUDA:
         CUDA_CALL(cudaMalloc(&ptr, size), ": size=" << size);
+        if (!async) {
```

Comment:
very minor: maybe for readability it's more natural to first handle `if (async)` then else branch

Reply:
Fixed.